src/graphql/core/queries.ts

```diff
@@ -0,0 +1,17 @@
+import { GT } from "../index"
+
+import Date from "../types/scalars/date"
+
+const QueryType = new GT.Object({
+  name: "Query",
+  fields: () => ({
+    currentTime: {
```

Comment:
why to create a mutation and a query for current time?

Reply:
This is just a placeholder starting point or the new core schema